GD32F4XX/standard_peripheral/Include/gd32f4xx_can.h

```diff
@@ -674,7 +674,7 @@ typedef enum
 #define CAN_FT_REMOTE                      ((uint32_t)0x00000002U)      /*!< remote frame */
 
 /* CAN timeout */
-#define CAN_TIMEOUT                        ((uint32_t)0x0000FFFFU)      /*!< timeout value */
+#define __CAN_TIMEOUT                      ((uint32_t)0x0000FFFFU)      /*!< timeout value */
```

Comment:
Please adjust references in the corresponding `_can.c` driver.

Reply:
Sorry, _can.c have the same use now.

Reply:
Same comment I made at db8c7aab. We should state at #8 first how do things to avoid conflicts.